scripts/release.sh

```diff
@@ -40,11 +40,8 @@ IMAGE_NAME="ghcr.io/software-mansion/smelter"
 docker pull "${IMAGE_NAME}:${COMMIT_HASH}"
 docker pull "${IMAGE_NAME}:${COMMIT_HASH}-web-renderer"
 
-docker tag "${IMAGE_NAME}:${COMMIT_HASH}" "${IMAGE_NAME}:${RELEASE_TAG}"
-docker tag "${IMAGE_NAME}:${COMMIT_HASH}-web-renderer" "${IMAGE_NAME}:${RELEASE_TAG}-web-renderer"
-
-docker push "${IMAGE_NAME}:${RELEASE_TAG}"
-docker push "${IMAGE_NAME}:${RELEASE_TAG}-web-renderer"
+docker buildx imagetools create -t "${IMAGE_NAME}:${RELEASE_TAG}" "${IMAGE_NAME}:${COMMIT_HASH}"
```

Comment:
Is the docker pull above needed? I assumed that it only fetches image for one arch, so this command must not rely on what actual is fetched locally. (just guessing here)

Reply:
You're correct. The pull is not needed here. Fixed